presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java

```diff
@@ -297,6 +346,14 @@ private void testStrPosAndPosition(String string, String substring, Long expecte
         assertFunction(format("POSITION(%s in %s)", substring, string), BIGINT, expected);
     }
 
+    private void testStrReversePosAndPosition(String string, String substring, Long expected)
+    {
+        string = (string == null) ? "NULL" : ("'" + string + "'");
```

Comment:
This helper function is unnecessary. Directly use `assertFunction` at call site.

Reply:
Done.